packages/stacks/src/validation/address-validation.ts

```diff
@@ -33,11 +29,5 @@ export function isValidAddressChain(address: string, chainId: ChainId) {
 }
 
 export function validatePayerNotRecipient(senderAddress: string, recipientAddress: string) {
-  if (!isValidStacksAddress(senderAddress) || !isValidStacksAddress(recipientAddress)) {
```

Comment:
Nice refactor 👍  are the `stacks` and `btc` address validation steps just not needed now or were they bothering you?

Reply:
Those are redundant, the outcome will be the same without those checks. Helpers like `validatePayerNotRecipient` or `isValidAddressChain` can already assume the input is a valid address. 
Forgot the botched tests, will fix.